### PR TITLE
Fixing parent-child bug

### PR DESCRIPTION
**Pull request by agutierrez-eb**
In this PR, the parent-child bug is fixed. Now we added a raw query in the new script only for the childs, and generate a plain dict, but this is temporary, because we will create objects per result in the new query. 
Also, added tests.

### CODE REVIEW

File: invoicing_app/tests/tests.py [hunk 1]
```diff
@@ -604,13 +608,16 @@ def test_childs(self, call_new, call_old):
...
+
         my_order_child_3 = OrderFactory.build(
             event=my_event_child_3
         )
         my_order_child_3.save()
         # ------------------------------------------------ #
+
         self.my_command.handle(**self.options)
         self.my_command_new.handle(**self.options)
+        # import ipdb; ipdb.set_trace()
```
Comment (fgomez-eb) [inline invoicing_app/tests/tests.py]: here u leave a unnecessary comment

File: invoicing_app/management/commands/generate_tax_receipts_new.py [hunk 2]
```diff
@@ -193,22 +198,82 @@ def handle(self, **options):
             total_taxable_amount_with_tax_amount=Sum('mg_fee'),
             total_tax_amount=Sum('eb_tax'),
             payment_transactions_count=Count('event'),
-        ).iterator()
```
Comment (fgomez-eb) [inline invoicing_app/management/commands/generate_tax_receipts_new.py:196]: i think the iterator stay.

File: invoicing_app/management/commands/generate_tax_receipts_new.py [hunk 3]
```diff
@@ -19,9 +19,14 @@
 
 from invoicing_app.models import PaymentOptions, Order
 
+from django.db import connection
+from django.db.models.query import ValuesQuerySet
```
Comment (fgomez-eb) [inline invoicing_app/management/commands/generate_tax_receipts_new.py]: u use this import?

File: invoicing_app/management/commands/generate_tax_receipts_new.py [hunk 4]
```diff
@@ -19,9 +19,14 @@
...
+
 DATE_FORMAT = '%Y-%m-%dT%H:%M:%SZ'
 
 
+# django.db.models.query.ValuesQuerySet
```
Comment (fgomez-eb) [inline invoicing_app/management/commands/generate_tax_receipts_new.py]: remove comments

File: invoicing_app/management/commands/generate_tax_receipts_new.py (hunk 2 above)
Comment (fgomez-eb) [inline invoicing_app/management/commands/generate_tax_receipts_new.py:196]: i think the iterator have to stay

File: invoicing_app/management/commands/generate_tax_receipts_new.py (hunk 3 above)
Comment (agutierrez-eb) [reply invoicing_app/management/commands/generate_tax_receipts_new.py]: Conection is used in the cursor for the query raw. ValuesQuerySet is not used

File: invoicing_app/tests/tests.py (hunk 1 above)
Comment (agutierrez-eb) [reply invoicing_app/tests/tests.py]: You're right!

File: invoicing_app/management/commands/generate_tax_receipts_new.py (hunk 2 above)
Comment (agutierrez-eb) [reply invoicing_app/management/commands/generate_tax_receipts_new.py:196]: You're right

**Review by ghidalgo-eb — DISMISSED** [review]

**Review by fgomez-eb — APPROVED** [review]
LGTM

**Pull request merged**